umm-spec-lib/src/cmr/umm_spec/iso19115_2_util.clj

```diff
@@ -13,6 +13,14 @@
 (def short-name-xpath
   "gmi:identifier/gmd:MD_Identifier/gmd:code/gco:CharacterString")
 
+;; relative to gmi:childOperation
+(def campaign-xpath
+  "gmi:MI_Operation/gmi:identifier/gmd:MD_Identifier/gmd:code/gco:CharacterString") 
+
+(def start-end-date-xpath
```

Comment:
"gmi:description/gco:CharacterString" could be many things. but start-end-date-xpath could only be one. It's the same with these other paths too. like short-name-xpath, long-name-xpath and other xpaths. None of them imply the meaning of the xpath.   If you really want to change it, what do you recommend that we use?